mongo/integration/mtest/options.go

```diff
@@ -65,14 +67,20 @@ func (r *RunOnBlock) UnmarshalBSON(data []byte) error {
 		Topology         []TopologyKind           `bson:"topology"`
 		Topologies       []TopologyKind           `bson:"topologies"`
 		ServerParameters map[string]bson.RawValue `bson:"serverParameters"`
+		Auth             *bool                    `bson:"auth"`
+		Extra            map[string]interface{}   `bson:",inline"`
 	}
 	if err := bson.Unmarshal(data, &temp); err != nil {
 		return fmt.Errorf("error unmarshalling to temporary RunOnBlock object: %v", err)
 	}
+	if len(temp.Extra) > 0 {
```

Comment:
I noticed this was missing and added it for consistency with the other types.

Reply:
Nice. The `Extra` field for catching extra BSON fields is cool.